<ui version="4.0" >
 <class>ScreenShare</class>
 <widget class="QWidget" name="ScreenShare" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>176</width>
       <height>144</height>
      </size>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="2" >
       <widget class="QToolButton" name="recvScreenShareButton" >
        <property name="text" >
         <string>接收共享文档</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QToolButton" name="openFileButton" >
        <property name="text" >
         <string>打开文档</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QScrollArea" name="scrollArea" >
          <property name="widgetResizable" >
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>830</width>
             <height>513</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QToolButton" name="shareButton" >
        <property name="text" >
         <string>共享文档</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc" />
 </resources>
 <connections/>
</ui>
